src/commons/BrowserUtils.ts

```diff
@@ -706,6 +708,36 @@ export namespace BrowserUtils {
     }
   }
 
+  /**
+   * Change size of browser window
+   *    If only one parameter is provided the second will stay as is.
+   *    If no parameter provided there will be no change.
+   * @param requestedWidth - Width (px)
+   * @param requestedHeight - Height (px)
+   */
+  export function setWindowSize(
+    requestedWidth?: number,
+    requestedHeight?: number
+  ): void {
+      if(requestedWidth === undefined && requestedHeight === undefined ) {
+          Reporter.debug("No new window size requested, aborting window size action.");
+
+          return;
+      }
+    const size: Size = getWindowSize();
+    size.width = requestedWidth === undefined ? size.width : requestedWidth;
+    size.height = requestedHeight === undefined ? size.height : requestedHeight;
+    Reporter.debug(`size >> ${JSON.stringify(size)}`);
+    browser.windowHandleSize(size);
+  }
+
+  /**
+   * Get current browser window size returns Size obj {width : number, height : number}
+   */
+  export function getWindowSize(): Size {
```

Comment:
incorrect function name - getWindowSize returns viewport size. it's not the same.
+ add Reporter.debug message

you might try to use:
window.innerWidth
window.innerHeight

Reply:
done